### PR TITLE
Use relative path for generating HTML Hurl file

### PR DESCRIPTION
**Pull request by fabricereix**

### CODE REVIEW

File: packages/hurl/src/report/mod.rs [hunk 1]
```diff
@@ -225,13 +225,17 @@ fn create_html_result(result: HurlResult) -> html::Element {
...
+        None => result.filename.as_str(),
+        Some(value) => value,
+    };
     html::Element::NodeElement {
         name: "tr".to_string(),
         attributes: vec![
             html::Attribute::Class(status.clone()),
             html::Attribute::Data("duration".to_string(), result.time_in_ms.to_string()),
             html::Attribute::Data("status".to_string(), status.clone()),
-            html::Attribute::Data("filename".to_string(), result.filename.clone()),
```
Comment (jleverenz) [inline packages/hurl/src/report/mod.rs:234]: I think line 242 below needs a similar update to correctly link the href.

Comment (fabricereix) [reply packages/hurl/src/report/mod.rs:234]: you're right!

Comment (fabricereix) [reply packages/hurl/src/report/mod.rs:234]: I keep it in draft until I add more tests.
Thanks

**Review by jleverenz — APPROVED** [review]
lgtm, not sure what's going on with the failed ci check though

**Comment by fabricereix** [comment]
We have often issues with the job in circle-ci (Testing ARM architecture).
We are going to try to migrate it to a github runner.

**Pull request merged**